docker-compose.yml

```diff
@@ -57,9 +57,12 @@ services:
     profiles:
       - nondefault
     volumes:
-      - ./config/nginx/:/etc/nginx/
+      - ./config/nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+      - ./config/nginx/nginxconfig.io/:/etc/nginx/nginxconfig.io/:ro
+      - ./config/nginx/sites-enabled/:/etc/nginx/sites-enabled/:ro
```

Comment:
Enumerating specific files/directories allows us to do this above: `include /etc/nginx/mime.types;`. Previously, we had been overwriting that file, by overwriting the whole `/etc/nginx/` directory with our own config (which doesn't include `mime.types`).

Reply:
(`:ro` means "read only".)

Reply:
> Enumerating specific files/directories allows us to do this above: `include /etc/nginx/mime.types;`. Previously, we had been overwriting that file, by overwriting the whole `/etc/nginx/` directory with our own config (which doesn't include `mime.types`).

I think that this is wrong; see https://github.com/davidrunger/david_runger/pull/4890 .